Update QNAP integration (#17780)
home-assistant/home-assistant.github.io: source/_integrations/qnap.markdown

ha_iot_class: Local Polling
ha_codeowners:
  - '@colinodell'
  - '@cyr-ius'

> ha_domain: qnap

Reviewer: - Missing `ha_config_flow`.

Does it have discovery now as well? In that case, that needs to be added as well.

ha_platforms:
  - sensor
---

> This `qnap` sensor allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).

Reviewer: ```suggestion
The QNAP integration allows getting various statistics from your [QNAP NAS](https://www.qnap.com/en-us/).
```

{% include integrations/config_flow.md %}

## Integration Entities

The QNAP integration will add the following sensors.

> Sensors:
> - system: status, system temperature
> - cpu : usage, temperature
> - memory : usage, free, used, total
> - volume : usage, used
> - folders : usage, used
> - drives: smart status, usage
> - networks uplink : bandwidth, status

Reviewer: ```suggestion
Sensors:

- System: status, system temperature
- CPU: usage, temperature
- Memory: usage, free, used, total
- Volume: usage, used
- Folders: usage, used
- Drives: smart status, usage
- Network: bandwidth, status
```

> ### SSL and Self-signed certificates

Reviewer: This section needs adjustment as well, as it references to YAML configuration that has been removed.
